src/executor/tokio.rs

```diff
@@ -31,7 +31,7 @@ impl AgnostikExecutor for TokioExecutor {
         F: Future + Send + 'static,
         F::Output: Send + 'static,
     {
-        let handle = self.0.lock().unwrap().spawn(future);
+        let handle = tokio::task::spawn(future);
```

Comment:
I would rather not change that to the global method because we use the `Runtime` methods for every other operation too and we can't replace all methods with a global method because there's no `block_on` or `run` method without a `Ruuntime` instance. So I think it's better to stick with the `Runtime` instance

Reply:
> we use the Runtime methods for every other operation too

That's not true, the global `spawn_blocking` and `spawn_local` are also used. The reason for this change is because I couldn't get task spawning to work otherwise. It simply never returns.

Reply:
> That's not true, the global spawn_blocking and spawn_local are also used.
Whoops, totally forgot that. 

Yea then the change is fine :+1: